data/libs/systems/vip.lua

```diff
@@ -0,0 +1,62 @@
+-- Bonus mount and outfits
+local outfits = {}
+local mounts = {}
+
+function Player.onRemoveVip(self)
```

Comment:
Wouldn't it be better to check if the player has VIP before withdrawing?

`if player:getVipTime() < os.time() then return end
`

Reply:
> Wouldn't it be better to check if the player has VIP before withdrawing?
> 
> `if self:getVipTime() < os.time() then return end `

I wouldn't like to make changes to the code logic in this pull request, I think it's better to open an issue or a new pull request.